libpip2pi/commands.py

```diff
@@ -373,7 +373,7 @@ def _dir2pi(option, argv):
             shutil.copy2(pkg_filepath, symlink_target)
 
         if pkg_name not in processed_pkg:
-            pkg_index += "<a href='%s/'>%s</a><br />\n" %(
+            pkg_index += "<a href='%s/index.html'>%s</a><br />\n" %(
```

Comment:
This may fail if you have a server-generated  index that's under some other name?

Reply:
you are right, also this change will conflict with the "standard"(https://www.python.org/dev/peps/pep-0503/)

actually my initial intention was to support deploying pypi package repository by using "object storage service" which does't support "directory index".